pylint_odoo/test_repo/broken_module/model_view2.xml

```diff
@@ -104,6 +104,14 @@
             <field name="name">view.model.form80</field>
             <field name="model">test.model</field>
         </record>
-
+        <!-- Replace with low priority wo field -->
+        <record id="view_model_form90" model="ir.ui.view">
+            <field name="name">view.model.form90</field>
+            <field name="model">test.model</field>
+            <field name="priority">10</field>
+            <field name="arch" type="xml">
+                 <label for="name" position="replace"/>
```

Comment:
Nice. I'd provide also another view with another position, to ensure only replace is taken into account

Reply:
Done